app/assets/src/components/views/report/ReportTable/DetailCells.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+
+function DetailCells({ parent, openTaxonModal }) {
```

Comment:
Our convention is to use the format: `const DetailCells = ({parent, openTaxonModal}) => {...}`